java/team25core/ColorSensorTask.java

```diff
@@ -4,14 +4,12 @@
 import com.qualcomm.robotcore.hardware.ColorSensor;
 import com.qualcomm.robotcore.hardware.DeviceInterfaceModule;
 
-import opmodes.NeverlandAutonomousConstants;
-
 /**
  * Created by katie on 12/14/15.
  */
 public class ColorSensorTask extends RobotTask {
 
-    public final static float BEACON_THRESHOLD = NeverlandAutonomousConstants.COLOR_THRESHOLD;
+    public final static float BEACON_THRESHOLD = 287;
```

Comment:
This might be something that is specific to the application, and so perhaps we want to force anyone using this task to pass this value in through the constructor?